.lando.yml

```diff
@@ -0,0 +1,69 @@
+name: exactions
+proxy:
+  django:
+    - exactions.lndo.site:8000
+  nodejs:
+    - exactions-node.lndo.site:3000
+
+services:
+  django:
+    type: python:custom
+    overrides:
+      image: exactions_local
+      build:
+        context: ./backend
+        dockerfile: ./Dockerfile
+        args:
+          BUILD_ENVIRONMENT: local
+      user: root
+      environment:
+        PYTHONPATH: /app/backend
+        DJANGO_DEBUG: "True"
+        DATABASE_URL: postgres://postgres:@database.exactions.internal:5432/django
+    ssl: true
+    scanner: false
+
+  database:
+    type: postgres:14
+    creds:
+      user: postgres
+      password: ""
+      database: django
+
+  nodejs:
+    type: "node:8"
+    run:
+      - mkdir -p /home/node/.local
+      - cd /app/frontend && npm i
```

Comment:
Let's tack on `&& npm run build` so the site works on the first call to `lando runserver`.

Reply:
done